extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -99,11 +105,20 @@ export class Bind {
     this.resources_ = resourcesForDoc(ampdoc);
 
     /**
+     * True if all bindings in the document have been scanned and parsed.
+     * @private {boolean}
+     */
+    this.initialized_ = false;
+
+    /**
      * True if a digest is triggered before scan for bindings completes.
      * @private {boolean}
      */
     this.digestQueuedAfterScan_ = false;
 
+    /** @const @private {boolean} */
+    this.workerExperimentEnabled_ = isExperimentOn(this.win_, 'web-worker');
```

Comment:
Looks like [web workers are available](http://caniuse.com/#feat=webworkers) for all browsers we care about, so I don't think we should bother supporting the fallback case. It's still useful to check this in `amp-worker.js` so we can reject `invokeWebWorker` with a meaningful error message.